### ISSUE TITLE
Node.warn(code, message) form has been deprecated, use Node.warn(warning_instance) instead

### ISSUE THREAD

**Issue by The-Compiler** [ionelmc/pytest-benchmark#124]
With pytest 3.9.2 and pytest-bdd 3.0.0 I get this:

```
________________ ERROR at teardown of test_match_benchmark[webkit] _________________

tp = <class '_pytest.warning_types.RemovedInPytest4Warning'>, value = None
tb = None

    def reraise(tp, value, tb=None):
        try:
            if value is None:
                value = tp()
            if value.__traceback__ is not tb:
                raise value.with_traceback(tb)
>           raise value

.tox/py37-pyqt511/lib/python3.7/site-packages/six.py:693: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
.tox/py37-pyqt511/lib/python3.7/site-packages/six.py:693: in reraise
    raise value
.tox/py37-pyqt511/lib/python3.7/site-packages/six.py:693: in reraise
    raise value
.tox/py37-pyqt511/lib/python3.7/site-packages/pytest_benchmark/fixture.py:252: in _cleanup
    warner=self._warner, suspend=True)
.tox/py37-pyqt511/lib/python3.7/site-packages/pytest_benchmark/logger.py:40: in warn
    warner(code=code, message=text)
.tox/py37-pyqt511/lib/python3.7/site-packages/six.py:693: in reraise
    raise value
.tox/py37-pyqt511/lib/python3.7/site-packages/six.py:693: in reraise
    raise value
.tox/py37-pyqt511/lib/python3.7/site-packages/six.py:693: in reraise
    raise value
.tox/py37-pyqt511/lib/python3.7/site-packages/pytest_benchmark/fixture.py:252: in _cleanup
    warner=self._warner, suspend=True)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <pytest_benchmark.logger.Logger object at 0x7f5e2c139400>
code = 'BENCHMARK-U1', text = 'Benchmark fixture was not used at all in this test!'
warner = <bound method Node.warn of <Function 'test_show_benchmark[webkit]'>>
suspend = True, fslocation = None

    def warn(self, code, text, warner=None, suspend=False, fslocation=None):
        if self.verbose:
            if suspend and self.capman:
                self.capman.suspendcapture(in_=True)
            self.term.line("")
            self.term.sep("-", red=True, bold=True)
            self.term.write(" WARNING: ", red=True, bold=True)
            self.term.line(text, red=True)
            self.term.sep("-", red=True, bold=True)
            if suspend and self.capman:
                self.capman.resumecapture()
        if warner is None:
            warner = self.pytest_warn
        if fslocation and self.pytest_warn_has_fslocation:
            warner(code=code, message=text, fslocation=fslocation)
        else:
>           warner(code=code, message=text)
E           _pytest.warning_types.RemovedInPytest4Warning: Node.warn(code, message) form has been deprecated, use Node.warn(warning_instance) instead.
```

**Comment by The-Compiler** [comment]
Yup, the current git master seems to work fine here - any chance for a new release? :wink:

**Comment by ionelmc** [comment]
Soon :)

**Comment by ofek** [comment]
Master appears to be passing with 4.1. Anything else blocking the release?

**Comment by ionelmc** [comment]
Released 3.2.0 now.
